src/app/globals.css

```
@@ -200,29 +205,154 @@ body * {
-  background-color: #f3f4f6 !important;
-  color: #eb5757 !important;
-  padding: 0.2rem 0.4rem !important;
-  border-radius: 4px !important;
-  font-family: monospace !important;
+  background-color: #f6f8fa !important;
+  color: #c7254e !important;
+  padding: 0.15rem 0.4rem !important;
+  border: 1px solid #e5e7eb !important;
+  border-radius: 6px !important;
+  font-family:
+    "SFMono-Regular", Consolas, "Liberation Mono", Menlo, monospace !important;
```

Stylelint (17.3.0)

[error] Unexpected quotes around "SFMono-Regular" (font-family-name-quotes)

(font-family-name-quotes)